Changelog.md

```diff
@@ -23,6 +23,7 @@ Bugfixes:
  * View/Pure Checker: Properly detect state variable access through base class.
  * Yul analyzer: Check availability of data objects already in analysis phase.
  * Yul Optimizer: Fix an issue where memory-accessing code was removed even though ``msize`` was used in the program.
+ * ABI decoder: Always create copies of dynamic memory arrays during decoding.
```

Comment:
Please move it up the list (alphabetically sorted)

Reply:
Not sure why I didn't :-).